build/package.sh

```diff
@@ -1,30 +1,37 @@
 #!/bin/bash
-bindir=$1
+set -e
+binroot=$1
 artifactsdir=$2
-unamestr=`uname`
 
 if [[ -z "${TRAVIS_TAG}" ]]; then
     echo "TRAVIS_TAG is missing... Release should be skipped..."
     TRAVIS_TAG="notags"
 fi
 
-if [[ "$unamestr" == "Linux" ]]; then
-	osname="linux"
-elif [[ "$unamestr" == "Darwin" ]]; then
-	osname="osx"
-else
-    echo "OS $unamestr is not yet supported"
-    exit -1
+if [[ ! -d $binroot ]]; then
+    echo "Please run the script from project root..."
+	exit -1
 fi
 
 if [[ ! -d $artifactsdir ]]; then
     mkdir $artifactsdir
 fi
 
-pkgFile="docker-credential-acr-${osname}-amd64.tar.gz"
-pushd ${bindir}
-tar czf ${pkgFile} *
-popd
-mv ${bindir}/$pkgFile ${artifactsdir}
-
-echo "Packaged in ${pkgFile}"
+for osname in $(ls $binroot) ; do
+    bindir="$binroot/$osname"
+    if [[ -d $bindir ]]; then
+        echo "Packaging for ${osname}"
+        pkgName="docker-credential-acr-${osname}-amd64"
```

Comment:
Set "amd64" in a variable and append the variable here. We may build this helper for other architectures in the future and we want to have this ready to go.